bytes.go

```diff
@@ -77,30 +73,33 @@ func ToMegabytes(s string) (uint64, error) {
 // GB = G = GiB = 1024 * M
 // TB = T = TiB = 1024 * G
 func ToBytes(s string) (uint64, error) {
-	parts := bytesPattern.FindStringSubmatch(strings.TrimSpace(s))
-	if len(parts) < 3 {
-		return 0, invalidByteQuantityError
-	}
+	s = strings.TrimSpace(s)
+	s = strings.ToUpper(s)
 
-	value, err := strconv.ParseFloat(parts[1], 64)
-	if err != nil || value <= 0 {
-		return 0, invalidByteQuantityError
-	}
+	i := strings.IndexFunc(s, func(v rune) bool {
+		return unicode.IsLetter(v)
```

Comment:
I think you can just pass `unicode.IsLetter` instead of wrapping it in a function.

Reply:
yep, you're right